common/gl_util/lib/stb_image.h

```diff
@@ -4120,7 +4120,7 @@ static int stbi__zhuffman_decode_slowpath(stbi__zbuf *a, stbi__zhuffman *z)
    if (s >= 16) return -1; // invalid code!
    // code size is s, so:
    b = (k >> (16-s)) - z->firstcode[s] + z->firstsymbol[s];
-   if (b >= (int) sizeof (z->size)) return -1; // some data was corrupt somewhere!
+   if (size_t(b) >= sizeof (z->size)) return -1; // some data was corrupt somewhere!
```

Comment:
Please revert.

Reply:
I think this is better. Extending an `int` to a `size_t` is safer than `size_t` to `int`.